control/trajectory_follower/obstacle_collision_checker/src/obstacle_collision_checker_node/obstacle_collision_checker.cpp

```diff
@@ -0,0 +1,261 @@
+// Copyright 2020 Tier IV, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <boost/geometry.hpp>
```

Comment:
Replace <> with "" except for stl header.

Reply:
I'll replace <>.
I would like to know why. Where do I find that information?
Isn't "boost" something like "stl"?